src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useContext } from 'react';
+import cn from 'classnames';
+import { TodosFilter } from '../types/TodosFilter';
+import { TodoContext } from '../Context/TodoContext';
+
+const setFilterHref = (filter: TodosFilter) => {
+  switch (filter) {
+    case TodosFilter.Active:
+      return '#/active';
+
+    case TodosFilter.Completed:
+      return '#/completed';
+
+    case TodosFilter.All:
+    default:
+      return '#/';
+  }
+};
+
+const setFilterDataCy = (filter: TodosFilter) => {
+  switch (filter) {
+    case TodosFilter.Active:
+      return 'FilterLinkActive';
```

Comment:
You do the same checks so you can return an object and one function is enough in this case.

```javascript
    case TodosFilter.Active:
      return {
        href: '#/active',
        dataCy: 'FilterLinkActive',
      };
```